app/src/main/java/org/thoughtcrime/securesms/onboarding/messagenotifications/MessageNotifications.kt

```diff
@@ -112,13 +121,32 @@ private fun NotificationRadioButton(
                     RoundedCornerShape(8.dp)
                 ),
         ) {
-            Column(modifier = Modifier
-                .padding(horizontal = LocalDimensions.current.smallSpacing,
-                    vertical = LocalDimensions.current.xsSpacing)
-                ) {
-                Text(stringResource(title), style = LocalType.current.h8)
+            Column(
+                modifier = Modifier.padding(horizontal = LocalDimensions.current.smallSpacing, vertical = LocalDimensions.current.xsSpacing)) {
+                Text(
+                    stringResource(title),
+                    style = LocalType.current.h8
+                )
+
+                // If this radio button is the one for slow mode notifications then substitute the app name..
+                if (explanation == R.string.notificationsSlowModeDescription) {
```

Comment:
This defeats the point of making reusable components in Compose. The `NotificationRadioButton` should be agnostic about its content. It might even be reused in other parts of the app.
The composable calling `NotificationRadioButton` is responsible for passing the right content directly.

Reply:
Good point - I've created an overridden version of `NotificationRadioButton` that takes the pre-substituted title and explanation text rather than performing the substitution within the radio button's creation.